packages/service/src/domain/commands/processors/playback-current-command.ts

```diff
@@ -0,0 +1,73 @@
+import { Command } from '@service/domain/commands/model/command';
+import { CommandProcessingResponse, CommandProcessingResponseBuilder } from '@service/domain/commands/model/command-processing-response';
+import { CommandParameters, CommandParametersBuilder, CommandProcessor } from '@service/domain/commands/model/command-processor';
+import { RegisterCommand } from '@service/domain/commands/registry/register-command';
+import { PlayerStateRequestEvent } from '@service/event-stream/model/events';
+import { PlayerEventStream } from '@service/event-stream/player-event-stream';
+import { PlayerState } from '@service/domain/player-state/player-state';
+import { Service } from 'typedi';
+import { Song } from '@service/lib';
+
+@RegisterCommand
+@Service()
+class PlaybackCurrentCommand extends CommandProcessor {
+  constructor(private playerEventStream: PlayerEventStream) {
+    super();
+  }
+
+  async execute(command: Command): Promise<CommandProcessingResponse> {
+    const includeQueue = command.rawArgs === 'queue';
+
+    const playerStateRequest: PlayerStateRequestEvent = { id: 'PlayerStateRequestEvent' };
+    const currentPlayerState: PlayerState = await this.playerEventStream.sendToPrimaryPlayer(playerStateRequest);
+
+    const text = this.buildMessage(
+      currentPlayerState.currentlyPlaying?.song ?? null,
+      currentPlayerState.queue ?? [],
+      includeQueue,
+    );
+
+    return new CommandProcessingResponseBuilder()
+      .fromMarkdown(text)
+      .build();
+  }
+
+  private buildMessage(current: Song | null, queue: Song[], includeQueue: Boolean): string {
+    const queueSongsTitles = includeQueue ? queue.map((song) => `  - ${song.name.replace(/(.{50})..+/, '$1…')}`) : [];
+    const queueTitle = !queueSongsTitles.isEmpty() ? 'W kolejce:' : 'Playlista jest pusta';
+    const currentSongName = current?.name ?? '';
+
+    const text = [
+      currentSongName ? 'Obecnie gramy:' : 'Obecnie nic nie gramy.',
```

Comment:
W oddzielnym PRze warto zrobić że jak obecnie coś gramy, to jeszcze dać link do YouTube